front/app/components/FormBuilder/components/FormFields/FieldElement.tsx

```diff
@@ -154,16 +161,24 @@ export const FieldElement = (props: Props) => {
                   mb="auto"
                   color="grey800"
                 >
-                  <T value={field.title_multiloc} />
-                  {!isDeleteEnabled && (
```

Comment:
Realized I did have the lock icon in, but I forgot to change the conditional check after updating to the new constraints :sweat_smile: